{{cookiecutter.project_slug}}/.github/workflows/docs.yml

```diff
@@ -0,0 +1,48 @@
+name: Documentation
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  docs:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout source
+        uses: actions/checkout@v4
+
+      - name: Cache tox
+        uses: actions/cache@v4
+        with:
+          path: .tox
+          key: tox-${{ '{{' }} hashFiles('pyproject.toml') {{ '}}' }}
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.x"
+          cache: "pip"
+          cache-dependency-path: "pyproject.toml"
+
+      - name: Install tox
+        run: python -m pip install tox
+      - name: Build HTML documentation with tox
+        run: tox -e docs
+{%- if cookiecutter.deploy_docs_to_github_pages %}
```

Comment:
`If not deploy_docs_to_github_pages`, should we instead dump the config commented out?
On one hand: it's a bunch WET of commented-out text which is bad practice and can easily fall out of sync. On the other, it's pretty nice to have if you change your mind.

Reply:
Yeah that could be a good idea in terms of allowing easier inclusion in future. I wonder in terms of going out of sync / avoiding being WET if there is some Jinja magic we can do to conditionally comment out the block we add if `deploy_docs_to_github_pages` is true (and probably add an additional explanatory comment instructing to uncomment to activate)? As @paddyroddy was not keen on the original commented out approach tagging him for comment.

Reply:
Oh, sorry I missed the original conversation.

Reply:
TIL WET. But yeah, I am really against commented bits of code. In my experience, they get ignored and fall out of sync very quickly. In general, only the original commenter is the one making sure it stays as intended, everyone else will ignore it.

Reply:
Okay seems like probably not worth looking at the conditional commenting of the block as an alternative so will mark this as resolved.